.pipelines/deploy-dev-env.yml

```diff
@@ -0,0 +1,81 @@
+# Azure DevOps Pipeline deploying a dev environement
+trigger: none
+pr: none
+
+variables:
+  - template: vars.yml
+
+jobs:
+  - job: "Deploy_new_shared_env"
+    pool:
+      vmImage: "ubuntu-latest"
+
+    steps:
+      - template: ./templates/template-setup-golang-env.yml
+        parameters:
+          gobin: ${{ variables.GOBIN }}
+          gopath: ${{ variables.GOPATH }}
+          goroot: ${{ variables.GOROOT }}
+          modulePath: ${{ variables.modulePath }}
+      - template: ./templates/template-setup-azure-tools.yml
+        parameters:
+          workingDirectory: ${{ variables.modulePath }}
+      - script: |
+          echo $(aro-v4-e2e-devops-spn) | base64 -d > aro-v4-e2e-devops-spn.json
```

Comment:
Is these visible in stdout? do we need set +x , -x wrappers?

Reply:
Secrets will be printed as `*****` in the console. 
We (MSFT + RH) are the only one having access to CI logs.

Was also planning to re add some `>/dev/null`